foreman/data_refinery_foreman/surveyor/sra.py

```diff
@@ -422,6 +433,14 @@ def _generate_experiment_and_samples(self, run_accession: str) -> (Experiment, L
             for key, value in harmonized_sample.items():
                 setattr(sample_object, key, value)
 
+            protocol_info, is_updated = self.update_sample_protocol_info(
+                [],
```

Comment:
Can you pass the kwarg name in here? It looks really weird to pass an empty array to as the first argument to a function

Reply:
Sure. I will.